modules/@apostrophecms/module/lib/events.js

```diff
@@ -4,6 +4,15 @@ module.exports = function(self) {
 
   return {
 
+    // Register an old event name to be accepted for a new event name.
```

Comment:
For now this only works for apostrophe core level events, but the API is designed so that if we need to we can extend it later to support aliases for module level events.